resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The accept header is invalid</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>1234567890
+                            <br>
+                            <br>
+                            <strong>Accept:</strong><br> application/vnd.hmrc.1.0
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Withdrawal charge has already been reported</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>0000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 
```

Comment:
Although this was brought in from the RAML, can this be removed? The maxLength is already specified in the regex already so this is redundant. maxLength should only be used when no regex is specified or the regex specified doesn't include it.

Reply:
Unfortunately I do not know what this comment relates too, as most of the comments relate to line 6400:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/13876870/218119684-770df204-fc90-4c84-9b9c-09587cdfb7d4.png">
And in most cases I need to search through the file to locate the issue, but here I have no any clue. If you could recheck and let me know would be great. Thanks.

Reply:
Did a quick sweep through the yaml to check where `maxLength` facet is used. It is used in two places [here](https://github.com/hmrc/lisa-api/blob/DDCE-3468/resources/public/api/conf/2.0/application.yaml#L9857) and [here](https://github.com/hmrc/lisa-api/blob/DDCE-3468/resources/public/api/conf/2.0/application.yaml#L10235). It is redundant to specify the `maxLength` facet when it is explicitly stated in the regex and so it can be removed.